package.json

```diff
@@ -6,20 +6,22 @@
     "scripts": {
         "build": "turbo build",
         "clean": "turbo clean",
+        "e2e": "cypress run --browser=chrome",
```

Comment:
i'd add `--headless` just to make sure it isn't using a headed version and causing issues.
There is allegedly a `--headless=new` to use the new version of chrome headless but it hasn't worked for me yet. Trying to find out about how it's used.